doc/keymap-format-text-v1.md

```diff
@@ -461,12 +809,398 @@ statement only defines the Group3 of a mapping:
+
+These actions perform different tasks on key press and on key release:
+
+<table>
+    <caption>
+        Effects of modifiers actions
+    </caption>
+    <!-- <thead> -->
+        <tr>
+            <th>Action</th>
+            <th>On key press</th>
+            <th>On key release</th>
+        </tr>
+    <!-- </thead> -->
+    <!-- <tbody> -->
+        <tr>
+            <th><code>SetMods</code></th>
+            <td>
+                <ul>
+                    <li>Adds modifiers to <em>depressed</em> modifiers</li>
+                </ul>
+            </td>
+            <td>
+                <ul>
+                    <li>
+                        Removes modifiers from <em>depressed</em> modifiers,
+                        provided that no other key which affects the same
+                        modifiers is logically down.
+                    </li>
+                    <li>
+                        If <code>clearLocks=yes</code> and no other key
+                        were operated simultaneously with this key,
+                        then the modifiers will be removed as well from
+                        the <em>locked</em> modifiers.
+                    </li>
+                </ul>
+            </td>
+        </tr>
+        <tr>
+            <th><code>LatchMods</code></th>
+            <td>
+                <ul>
+                    <li>Adds modifiers to <em>latched</em> modifiers.</li>
+                </ul>
+            </td>
+            <td>
+                <ul>
+                    <li>Removes modifiers from <em>latched</em> modifiers.</li>
+                    <li>
+                        If <code>clearLocks=yes</code> and no other key
+                        has been pressed since this key press, then the
+                        modifiers will be removed as well from the
+                        <em>locked</em> modifiers.
+                    </li>
+                    <li>
+                        If <code>latchToLock=yes</code> then the modifiers
+                        are added to the <em>locked</em> modifiers.
+                    </li>
+                </ul>
+            </td>
+        </tr>
+        <tr>
+            <th>
+                <code>LockMods</code>
+            </th>
+            <td>
+                <ul>
+                    <li>Adds modifiers to <em>depressed</em> modifiers.</li>
+                    <li>
+                        Toggle these modifiers in <em>locked</em> modifiers.</li>
+                    </li>
+                </ul>
+            </td>
+            <td>
+                <ul>
+                    <li>Removes modifiers from <em>depressed</em> modifiers.</li>
+                    <li><em>Locked</em> modifiers stay unchanged.</li>
+                </ul>
+            </td>
+        </tr>
+    <!-- </tbody> -->
+</table>
+
+TODO:
+
+### Group actions
+
+TODO:
+
+There are 3 group actions:
+- `SetGroup`: TODO
+- `LatchGroup`: TODO
+- `LockGroup`: TODO
+
+### Pointer actions
```

Comment:
All actions from here on are not supported by xkbcommon, i.e. they are parsed and validated but have no effect.

Reply:
Fixed